### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by jurko-gospodnetic**
Code cleanup done while working on another pull request and extracted here so it would not cloud up the original pull request.

### CODE REVIEW

File: testing/acceptance_test.py [hunk 1]
```diff
@@ -624,10 +624,9 @@ def join_pythonpath(*dirs):
         for p in search_path:
             monkeypatch.syspath_prepend(p)
 
-        os.chdir('world')
         # mixed module and filenames:
+        os.chdir('world')
```
Comment (RonnyPfannschmidt) [inline testing/acceptance_test.py]: this code<> commend swap looks unintended

**Comment by RonnyPfannschmidt** [comment]
overall good work on many many little details 👍

File: testing/acceptance_test.py (hunk 1 above)
Comment (jurko-gospodnetic) [reply testing/acceptance_test.py]: actually I thought it seemed correct - there are two pytest runs here and for each we set up a suitable cwd folder

easy enough to revert if you dislike it, I'm just curious as to what the intended meaning here was then

Comment (RonnyPfannschmidt) [reply testing/acceptance_test.py]: true, in full context it looks correct after all - before this change the lines where directly matching the code in question, but now its indeed marking the test sections instead

**Pull request merged**